tests/sentry/relay/snapshots/test_config/test_get_project_config/full_config/REGION.pysnap

```diff
@@ -101,6 +99,43 @@ config:
   groupingConfig:
     enhancements: eJybzDRxc15qeXFJZU6qlZGBkbGugaGuoeEEAHJMCAM
     id: newstyle:2023-01-11
+  metrics:
+    cardinalityLimits:
+    - id: unsupported
+      limit: 10000
```

Comment:
I don't have a lot of context, but does an "unsupported" metric mean we should drop it? If yes, should the limit be 0?

Reply:
`unsupported` is from the [Relay unsupported metric namespace](https://github.com/getsentry/relay/blob/master/relay-base-schema/src/metrics/mri.rs#L107-L127).

I have this as a fallback similar to what the [Python cardinality limiter does](https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_metrics/indexer/limiters/cardinality.py#L51-L57) as a "catch all".